src/js/components/VoterGuide/OrganizationPositionItem.jsx

```diff
@@ -11,33 +12,52 @@ export default class OrganizationPositionItem extends Component {
     let { ballot_item_display_name,
       kind_of_ballot_item,
       vote_smart_rating,
+      vote_smart_time_span,
       ballot_item_we_vote_id,
       ballot_item_image_url_https } = this.props.position;
 
-    return <li className="list-group-item">
+    return <li className="position-item">
           <StarAction we_vote_id={ballot_item_we_vote_id} type={kind_of_ballot_item} />
-        <Link className="linkLight"
-              to={"/candidate/" + ballot_item_we_vote_id }
+        <Link to={"/candidate/" + ballot_item_we_vote_id }
               onlyActiveOnIndex={false}>
           {/*<i className="icon-icon-add-friends-2-1 icon-light icon-medium" />*/}
-          {
-            ballot_item_image_url_https ?
-              <img
-                className="img-circle utils-img-contain"
-                style={{display: "block", paddingTop: "10px"}}
-                src={ballot_item_image_url_https}
-                alt="candidate-photo"/> :
-            <i className="icon-lg icon-main icon-icon-person-placeholder-6-1 icon-light"/>
+          { ballot_item_image_url_https ?
+            <img
+              className="position-item__avatar"
+              src={ballot_item_image_url_https}
+              alt="candidate-photo"/> :
+            <i className="icon-lg icon-main icon-icon-person-placeholder-6-1 icon-light position-item__avatar"/>
           }
-          &nbsp;<span> rates </span>
-          {ballot_item_display_name} {vote_smart_rating} %
-          <br />
-          {/*Running for {office_display_name}
-          <br />
-            Integer ut bibendum ex. Suspendisse eleifend mi accumsan, euismod enim at, malesuada nibh.
-            Duis a eros fringilla, dictum leo vitae, vulputate mi. Nunc vitae neque nec erat fermentum... (more)
-          <br />*/}
         </Link>
-        </li>;
+        <div className="position-item__content">
+          <Link to={"/candidate/" + ballot_item_we_vote_id }
+                onlyActiveOnIndex={false}>
+            <span className="position-rating__candidate-name">{ballot_item_display_name}</span>
+          </Link>
+          
+          {/* wasn't sure how to get this to work - Jeff */}
```

Comment:
Okay - here's a big chunk of brokenness. I just copied and pasted what I did in PositionItem.jsx but wasn't familiar enough with how to patch things in to get it to work. Because the explicit positions and Vote Smart ratings are being presented in different places, I would like to componentize them further, but probably would need to pair with someone or get help to do that.